packages/expo-image-picker/android/src/main/java/expo/modules/imagepicker/exporters/CompressionImageExporter.kt

```diff
@@ -12,9 +12,13 @@ import java.io.IOException
 
 class CompressionImageExporter(
   private val mImageLoader: ImageLoaderInterface,
-  private val mQuality: Int,
+  private val mQuality: Double,
```

Comment:
```suggestion
 mQuality: Double,
```
It doesn't have to be stored inside this class. Also, I think we should rename it to not start with `m`.